RetailSuiteAccounting/Integration/05_ManageCustomerSupplier.md

```diff
@@ -0,0 +1,97 @@
+[!!Fakturierung](RetailSuiteFaktBase)  
+[!!User Interface debtors/creditors](../UserInterface/02a_DebtorsCreditors.md)  
+
+# Manage the customer/supplier
+
+The *Accounting* module contains a function to create new customers and suppliers, and to enter all relevant information about them. This function is also available in the *Invoicing* module. This information is needed for both modules to operate.
+
+> [Info] Any customer/supplier information created, edited or deleted in the *Accounting* module will be also be created, edited or deleted in the *Invoicing* module, and vice versa. It may be necessary to refresh the page with the F5 key, though, to see the changes in the *Customer/supplier list* tab.
+
+
+## Create a customer/supplier
+
+You can create a customer/supplier that is not yet available in the system.
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](./01_RunAccountingWizard.md).
+- A valid fiscal year is created, see [Create a fiscal year](./04_ManageFiscalYear.md#create-a-fiscal-year).
+
+#### Procedure
+
+*Accounting > Select the fiscal year > Settings > Tab DEBTORS/CREDITORS > Tab Create customer/supplier*
+
+![Create a customer/supplier](../../Assets/Screenshots/RetailSuiteAccounting/Settings/CustomerSupplier/CreateCustomerSupplier.png "[Create a customer/supplier]")
+
+1. Click the *Create debtor/creditor* drop-down list in the *Address/Contact* sub-tab and select the appropriate option (**Debtor (customer)/Creditor (supplier)**).
+
+2. Enter a number for the customer/supplier in the *Debtor/creditor no.* field or leave it empty for the next free number.
+
+  > [Info] Make sure that the number entered is within the valid range (10000-69999 for customers and 70000-99999 for suppliers). Otherwise, a warning message will be displayed.
```

Comment:
```suggestion
  > [Info] If you enter a number, make sure that the number is within the valid range: 10000-69999 for customers and 70000-99999 for suppliers. Otherwise, a warning message will be displayed.
```